test/jit/tests/float-operations1.txt

```diff
@@ -0,0 +1,44 @@
+(module
+(func (export "test1") (result f32)
+    f32.const 3.3
+    f32.const 4.4
+    f32.max
+)
+(func (export "test2") (result f32)
+    f32.const 3.3
+    f32.const 4.4
+    f32.min
+)
+(func (export "test3") (result f32)
+    f32.const 4.0
+    f32.sqrt
+)
+(func (export "test4") (result f32)
+    f32.const 3.3
+    f32.ceil
+)
+(func (export "test5") (result f32)
+    f32.const 3.3
+    f32.floor
+)
+(func (export "test6") (result f32)
+    f32.const 3.3
+    f32.trunc
+)
+(func (export "test7") (result f32)
+    f32.const 3.3
+    f32.nearest
+)
+(func (export "test8") (result f32)
+    f32.const 3.3
+    f32.abs
+)
+(func (export "test9") (result f32)
+    f32.const -3.3
+    f32.abs
+)
+(func (export "test10") (result f32)
```

Comment:
There should be 64 bit tests, and nan tests as well. 
Comparison, binary and unary operation tests as well.
This patch adds a lot of opcodes.